llm_demo/orchestrator/vertexai_function_calling/functions.py

```diff
@@ -38,7 +42,16 @@
 
 search_amenities_func = generative_models.FunctionDeclaration(
     name="amenities_search",
-    description="Use this tool to search amenities by name or to recommend airport amenities at SFO. If top_k is not specified, default to 5",
+    description="""
+                Use this tool to search amenities by name or to recommended airport amenities at SFO.
+                If user provides flight info, use 'Get Flight' and 'Get Flights by Number'
```

Comment:
Are these the right tool names? Do we provide examples similar to the langchain tools?

Reply:
Thanks for catching this. I update these tool names. Checked langchain tools, it is updated there as well and it have the right name.